.github/CODEOWNERS

```
@@ -41,6 +41,7 @@
 airflow-core/src/airflow/ui/public/i18n/locales/ar/ @shahar1 @hussein-awala # + @ahmadtfarhan
 airflow-core/src/airflow/ui/public/i18n/locales/ca/ @jscheffl @bugraoz93 # + @ecodina
 airflow-core/src/airflow/ui/public/i18n/locales/de/ @jscheffl # + @TJaniF @m1racoli
+airflow-core/src/airflow/ui/public/i18n/locales/el/ @ashb # + @PApostol
```

Post mortem but still... @xBis7 as you helped review, was this a one-time-shot or would you join-in adding yourself to engaged translators (adding your GH handle to comment)? Like this:
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/el/ @ashb # + @PApostol @xBis7
```

---

I don't mind helping again with translations in the future. Feel free to add me. Thanks.

---

Easiest is... please open a small PR, add yourself and you even have a +1 in commit history :-D (I would not do anything else)

---

I opened a PR 

https://github.com/apache/airflow/pull/57128